src/registrar/migrations/0125_remove_domaininformation_submitter_and_more.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.10 on 2024-08-29 24:13
```

Comment:
Deletes submitter as a field on Domain Requests and Domain Info, fully deprecating it from the registrar